src/StreamJsonRpc/Reflection/MessageFormatterProgressTracker.cs

```diff
@@ -28,7 +28,7 @@ public class MessageFormatterProgressTracker
         /// <summary>
         /// Dictionary used to map the outbound request id to their progress id token so that the progress objects are cleaned after getting the final response.
         /// </summary>
-        private readonly Dictionary<long, object> requestProgressMap = new Dictionary<long, object>();
+        private readonly Dictionary<long, long> requestProgressMap = new Dictionary<long, long>();
```

Comment:
If this is changed to long, shouldn't the key in progressMap be changed to long as well?

Reply:
No. `progressMap` maps *incoming* messages while `requestProgressMap` maps `outgoing` messages. So while it is correct for `progressMap` to use `object` as the key since the ID could be a number or string, *this* map can correctly assume that the message ID is a `long` since we are always the author and always use longs.
So this was just a fix-up (unrelated to the rest of my changes) that I noticed I could make to yours while I was studying and copying from your class to make my own tracker class. :)